pkg/controller/backup/backup_controller.go

```diff
@@ -215,3 +218,36 @@ func (bkc *Controller) enqueueBackup(obj interface{}) {
 	}
 	bkc.queue.Add(key)
 }
+
+func validBackup(backup *v1alpha1.Backup) bool {
+	ns := backup.Namespace
+	name := backup.Name
+	if backup.Spec.BR == nil {
+		if backup.Spec.Cluster == "" {
+			glog.Infof("Missing Cluster config in spec of %s/%s", ns, name)
+			return false
```

Comment:
in this kind of situation I usually prefer to return `error`. With this approach the caller can decide to log the error, but also to support an interactive validation with a user the error message can actually be returned to the user. A test function can easily inspect error messages.

The logging level here seems like it should be error instead of into.

Reply:
If we return `error` here, this CR will be resynced later, actually, no need to resync the invalid CR, the best practice may be that we do validation in AA or webhook to reject the invalid CR.

For the logging level, I will change to `Error`, thanks!